build-image/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM golang:1.13.3-stretch
+FROM golang:1.13-buster
```

Comment:
I would suggest to specify the golang minor version to have reproducible builds. We can upgrade to the latest `1.13.6` tho.